paasta_tools/contrib/get_running_task_allocation.py

```diff
@@ -40,81 +36,9 @@ class TaskAllocationInfo(NamedTuple):
     host_ip: str
     git_sha: str
     config_sha: str
-    mesos_container_id: str  # Because Mesos task info does not have docker id
```

Comment:
might want to double-check if any of the splunk queries that we have will break if this field isn't present

Reply:
(it's probably safe since it's been null for ages, but you never know :p)